tests/core/notification/test_events_published.py

```diff
@@ -40,6 +41,11 @@ def identity(x):
 
 
 def test_event_published():
+    if _OrchestratorFactory._orchestrator is None:
+        _OrchestratorFactory._build_orchestrator()
+    _OrchestratorFactory._build_dispatcher()
+    Config.unblock_update()
+
```

Comment:
I wonder why this is needed.

Reply:
My bad. I believe this is redundant from the debugging process. I will remove it and try to rerun the tests.